### DISCUSSION TITLE
2.0-alpha1: auth.py vs checkin.py

### DISCUSSION THREAD

**Discussion by sebastian-luna-valero** [tdviet/fedcloudclient#192] [Q&A]
Hi,

With the new `auth.py` it looks like there is overlap with what's done in `checkin.py`.

Is there a plan to refactor to avoid duplication of code?

**Comment by tdviet** [comment]
Yes, I expect that only CLI functions remains in the checkin.py, all backend functions will be in auth.py.

There will be also keystone-auth module for obtaining and managing keystone tokens

**Comment by sebastian-luna-valero** [comment]
Ok, thanks!

If the plan is to just leave the CLI functions, would it be more appropriate to rename `checkin.py` to `token.py`?
